### PR TITLE
Extend working with Page description

### PR DESCRIPTION
**Pull request by michalmilc90**
Updated: Creating content - basic
Added: Working with Page

### CODE REVIEW

File: docs/working_with_page.md [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+    
+    - In the **Reveal** section, set the visibility to **Now** or plan it for future publish by clicking **Set date** and setting the date.
+    - In the **Hide** section, disable the option to hide the block by selecting **Never** or 
+    plan to hide it in future by clicking **Set date** and setting the date.
+    
+    !!! tip
+        Settings and function of custom-made blocks of content depend on their design. 
+        For details on adjusting and using them, contact your website administrator.
+    
+    ### Form block — adjusting settings
```
Comment (MagdalenaZuba) [inline docs/working_with_page.md]:
```suggestion
    ### Form block
```

File: docs/working_with_page.md [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+    Note that completing the settings of the Form block requires at least one Form Content item created.
+    
+    In the **Basic** tab, perform the following actions:
+    
+    - In the **Name** box, provide the name of the block that will appear on the Page.
+    - In the Form section, click **Select content**, navigate through the content and select a Form Content item to append it to the block.
+    
+    Click **Submit** to save the changes or **Discard** to cancel.
+    
+    ### Code block — adjusting settings
```
Comment (MagdalenaZuba) [inline docs/working_with_page.md]:
```suggestion
    ### Code block
```

File: docs/working_with_page.md [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+    
+    In the **Basic** tab, perform the following actions:
+    
+    - In the **Name** box, provide the name of the block that will appear on the Page.
+    - In the **Content** box, enter the HTML code for the content you want to display on the Page.
+    You can start with the part of the code that generally belongs to the `body` block.
+    
+    Click **Submit** to save the changes or **Discard** to cancel.
+    
+    ### Content List block — adjusting settings
```
Comment (MagdalenaZuba) [inline docs/working_with_page.md]:
```suggestion
    ### Content List block
```

File: docs/working_with_page.md [hunk 4]
```diff
@@ -0,0 +1,224 @@
...
+    - In the **Name** box, provide the name of the block that will appear on the Page.
+    - In the **Parent** section, click **Select content**, navigate through the content and select a folder containing Content items
+    to display on the list. Click **Confirm**.
+    - In the **Limit** section, enter a number or use the up and down buttons to set the limit of Content items to display.
+    - From the **Content Types to be displayed** list, select Content Type(s) to be displayed.
+    The block will display Content items of the selected Content Types that are in the selected parent folder.
+    
+    Click **Submit** to save the changes or **Discard** to cancel.
+    
+    ### Banner block — adjusting settings
```
Comment (MagdalenaZuba) [inline docs/working_with_page.md]:
```suggestion
    ### Banner block
```

File: docs/working_with_page.md [hunk 5]
```diff
@@ -0,0 +1,224 @@
...
+    
+    - In the **Name** box, provide the name of the block that will appear on the Page.
+    - In the **Location list** section, click **Select Content**, navigate through the content 
+    and add to the collection Content items of any Content Type you want.
+    All selected Content items appear in the **Confirmed items** box at the bottom of the window.
+    When done selecting, click **Confirm**.
+    
+    Click **Submit** to save the changes or **Discard** to cancel.
+    
+    ### Gallery block — adjusting settings
```
Comment (MagdalenaZuba) [inline docs/working_with_page.md]:
```suggestion
    ### Gallery block
```

File: docs/working_with_page.md [hunk 6]
```diff
@@ -0,0 +1,224 @@
...
+    
+    In the **Basic** tab, perform the following actions:
+    
+    - In the **Name** box, provide the name of the block that will appear on the Page.
+    - In the **Image** section, click **Select content**, navigate through the content and select an image to display.
+    - In the **URL** box, enter a URL to open when clicking the selected image.
+    
+    Click **Submit** to save the changes or **Discard** to cancel.
+    
+    ### Collection block — adjusting settings
```
Comment (MagdalenaZuba) [inline docs/working_with_page.md]:
```suggestion
    ### Collection block
```

File: docs/working_with_page.md [hunk 7]
```diff
@@ -0,0 +1,224 @@
...
+    In the **Basic** tab, perform the following actions:
+    
+    - In the **Name** box, provide the name of the block that will appear on the Page.
+    - In the **Folder** section, click **Select content**, navigate through the content, select a folder containing images to display and click **Confirm**. 
+    After submitting the settings, all images in the folder will appear in the Gallery block. 
+    Note that selecting a folder containing Content items other than images results in displaying only a link to the folder they are stored in.
+    
+    Click **Submit** to save the changes or **Discard** to cancel.
+    
+    ### Video block — adjusting settings
```
Comment (MagdalenaZuba) [inline docs/working_with_page.md]:
```suggestion
    ### Video block
```

File: docs/working_with_page.md [hunk 8]
```diff
@@ -0,0 +1,224 @@
...
+    
+    In the **Basic** tab, perform the following actions:
+    
+    - In the **Name** box, provide the name of the block that will appear on the Page.
+    - In the **Video** section, click **Select content**, navigate through the content, select a video to display in the block and click **Confirm**.
+    In the **Basic** tab you can preview the selected video before adding it to the Page.
+    
+    Click **Submit** to save the changes or **Discard** to cancel.
+    
+    ### RSS block — adjusting settings
```
Comment (MagdalenaZuba) [inline docs/working_with_page.md]:
```suggestion
    ### RSS block
```

File: docs/working_with_page.md [hunk 9]
```diff
@@ -0,0 +1,224 @@
...
+    
+    In the **Basic** tab, perform the following actions:
+    
+    - In the **Name** box, provide the name of the block that will appear on the Page.
+    - In the **URL** box, provide the URL for the RSS news feed.
+    - In the **Limit** box, enter a number or use the up and down buttons to set the limit of news items appearing in the block.
+    - In the **Offset** box, enter a number or use the up and down buttons to set the limit of featured news items appearing in the block.
+    
+    ### Content Scheduler block — adjusting settings
```
Comment (MagdalenaZuba) [inline docs/working_with_page.md]:
```suggestion
    ### Content Scheduler block
```

File: docs/working_with_page.md [hunk 10]
```diff
@@ -0,0 +1,224 @@
...
+    When done adjusting all settings, click **Submit** to save the changes or **Discard** to cancel.
+    
+    !!! tip
+        Access the timeline from the top toolbar to preview the scheduled publications or
+        click the button appearing to the right of the timeline to see all upcoming scheduled events.
+        Note that editing the Page is possible only when the preview is set to the current date and time.
+    
+    ![List of upcoming events](img/page_builder_toolbar_upcomingevents.png)
+    
+    ### Text block — adjusting settings
```
Comment (MagdalenaZuba) [inline docs/working_with_page.md]:
```suggestion
    ### Text block
```

File: docs/working_with_page.md [hunk 11]
```diff
@@ -0,0 +1,224 @@
...
+    
+    - In the **Name** box, provide the name of the block that will appear on the Page.
+    - In the **Content** box, enter text, images, videos, etc. using the Online Editor. 
+    For details, see [Editing Rich Text Fields](creating_content_basic.md#editing-rich-text-fields).
+    
+    When done editing, click **Submit** to save the changes or **Discard** to cancel.
+    
+    ## Publishing the Page
+    
+    If you are ready to publish the Page, in the menu, click **Publish**.
```
Comment (MagdalenaZuba) [inline docs/working_with_page.md]:
```suggestion
    If you are ready to publish the Page click **Publish** in the top-right corner.
```

File: docs/working_with_page.md [hunk 12]
```diff
@@ -0,0 +1,224 @@
...
+    To save it as a draft to finish editing it later, click **Save**.
```
Comment (MagdalenaZuba) [inline docs/working_with_page.md]:
```suggestion
    From the drop-down menu under **Publish** you can also publish the Page later, send it to review, save it as a draft or delete it.
```

File: docs/working_with_page.md [hunk 13]
```diff
@@ -0,0 +1,226 @@
+!!! enterprise
+
+    # Working with a Page
```
Comment (DominikaK) [inline docs/working_with_page.md]: Page title should be outside the "enterprise" box (it doesn't make a difference for rendering, but that's the convention we keep in dev-doc)

File: docs/working_with_page.md [hunk 14]
```diff
@@ -0,0 +1,226 @@
...
+        The Page Content Type contains a **Landing Page** Field Type which manages the zones and blocks.
+        Any Content item that has the **Landing Page** Field Type will behave like the Page Content Type.
+    
+    ## Getting started
+    
+    The Page Builder comes with a toolbar that enables you to perform advanced editing and previewing of the Page.
+    
+    ![Page Builder Toolbar](img/page_builder_toolbar.png)
+    
+    Using the toolbar buttons, you can perform the following actions:
```
Comment (DominikaK) [inline docs/working_with_page.md]: Good idea, this was missing 👍

File: docs/working_with_page.md [hunk 15]
```diff
@@ -0,0 +1,226 @@
...
+    
+    |Button|Description|
+    |------|-----------|
+    |![](img/page_builder_toolbar_editpreview.png)|Switch between previewing and editing the Page.|
+    |![](img/page_builder_toolbar_viewedit.png)|Switch between viewing the Page contents and the Page title and description.|
+    |![](img/page_builder_toolbar_devicestoggler.png)|Toggle through to see how the Page is rendered on different devices.|
+    |![](img/page_builder_toolbartimelinetoggler.png)|Access the timeline to preview the Page changes depending on the date or view the list of all upcoming scheduled events. For details, see [Timeline](publishing/advanced_publishing_options.md#timeline).|
```
Comment (DominikaK) [inline docs/working_with_page.md]: `Access the timeline...` This sentence is too convoluted. How about something like `Access the timeline to preview how the Page changes with time. You can also view...`

File: docs/working_with_page.md [hunk 16]
```diff
@@ -0,0 +1,226 @@
...
+    |![](img/page_builder_toolbarpopmenu.png)|Access menu enabling you to switch layout, see versions of the Page, manage translations, or delete the Page.|
+    
+    In a newly created Page you must first choose a layout which defines the available zones.
+    You can switch it later by using the following menu:
+    
+    ![Switching Page layouts](img/switch_layout.png)
+    
+    Applying a layout divides the Page into those zones that are place holders for the Content items.
```
Comment (DominikaK) [inline docs/working_with_page.md]:
```suggestion
    Applying a layout divides the Page into the defined zones. The zones are placeholders for Content items.
```

File: docs/working_with_page.md [hunk 17]
```diff
@@ -0,0 +1,226 @@
...
+    Now you are ready to add blocks of content to the Page.
+    
+    ## Adding blocks
+    
+    In the editor you can access a menu of **Elements** — a set of blocks of content that you can add to the zones of the Page.
+    Each block is unique in its function and enables you to personalize the content appearing on the Page.
```
Comment (DominikaK) [inline docs/working_with_page.md]: `personalize` is a domain term with a different meaning (personalized content, recommendations, etc.), so it's best to replace it.

File: docs/working_with_page.md [hunk 18]
```diff
@@ -0,0 +1,226 @@
...
+    eZ Enterprise comes with a set of ready-to-use Page blocks,
+    but [developers can add their own](https://doc.ezplatform.com/en/latest/guide/extending_page/#creating-page-blocks).
+    
+    ![Elements menu](img/elements_menu.png "Elements menu")
+    
+    The blocks provided with a clean installation of eZ Enterprise are:
+    
+    |Block|Description|
+    |-----|-----------|
+    |Form|Embeds a Form Content item that you select from the Content Structure.|
```
Comment (DominikaK) [inline docs/working_with_page.md]: Please add links to the descriptions for specific blocks below

File: docs/working_with_page.md [hunk 19]
```diff
@@ -0,0 +1,226 @@
...
+    |Code|Enables you to place text, links, images, etc. on your Page using custom HTML.|
+    |Content List|Displays Content items of a chosen Content Type (or Types) that are contained in a selected folder.|
+    |Banner|Displays an image Content item with a URL attached to it.|
+    |Collection|Displays a set of Content items you select manually from the Content structure.|
+    |Embed|Embeds a Content item of any Content Type on the Page.|
+    |Gallery|Displays all images contained in a selected folder.|
+    |Video|Embeds a video into the Page with standard playback controls.|
+    |RSS|Loads and displays news from RSS feeds (channels).|
+    |Content Scheduler|Displays Content items at a pre-define time.|
```
Comment (DominikaK) [inline docs/working_with_page.md]:
```suggestion
    |Content Scheduler|Displays Content items at a pre-defined time.|
```

File: docs/working_with_page.md [hunk 20]
```diff
@@ -0,0 +1,226 @@
...
+    You can access them by placing the cursor on the added block and clicking the cog button.
+    Use the same menu to remove the block from the Page by clicking the trash button.
+    
+    ![Elements settings](img/page_builder_elementsettings.png)
+    
+    Settings available for blocks are divided into three tabs — **Basic**, **Design**, and **Scheduler**.
+    The settings available in the **Basic** tab are tailored to the block's content.
+    The remaining tabs contain parameters common to all blocks provided with eZ Enterprise. 
+    Use them to modify the layout and visibility of a block.
+    Always ensure that you set all the required settings of a block of content to successfully publish the Page.
```
Comment (DominikaK) [inline docs/working_with_page.md]:
```suggestion
    You can't publish a Page if you have not set all the required settings of all blocks.
```

File: docs/working_with_page.md [hunk 21]
```diff
@@ -0,0 +1,226 @@
...
+        
+    In the **Design** tab, perform the following actions:
+    
+    - From the **View** list, select the block layout.
+    - *Optional:* In the **Class** box, define the class of the block
+    - *Optional:* In the **Style** box, add additional style formatting.
+    
+    In the **Scheduler** tab, define the visibility time of the block by performing the following actions:
```
Comment (DominikaK) [inline docs/working_with_page.md]: This is already covered in publishing/advanced_publishing_options/, so maybe just a link?

File: docs/working_with_page.md [hunk 22]
```diff
@@ -0,0 +1,226 @@
...
+    
+    Click **Submit** to save the changes or **Discard** to cancel.
+    
+    ### Code block
+    
+    In the **Basic** tab, perform the following actions:
+    
+    - In the **Name** box, provide the name of the block that will appear on the Page.
+    - In the **Content** box, enter the HTML code for the content you want to display on the Page.
+    You can start with the part of the code that generally belongs to the `body` block.
```
Comment (DominikaK) [inline docs/working_with_page.md]: Not sure about this sentence. Does it mean that you should not include `<head>` here?

File: docs/working_with_page.md [hunk 23]
```diff
@@ -0,0 +1,226 @@
...
+    ### RSS block
+    
+    In the **Basic** tab, perform the following actions:
+    
+    - In the **Name** box, provide the name of the block that will appear on the Page.
+    - In the **URL** box, provide the URL for the RSS news feed.
+    - In the **Limit** box, enter a number or use the up and down buttons to set the limit of news items appearing in the block.
+    - In the **Offset** box, enter a number or use the up and down buttons to set the limit of featured news items appearing in the block.
+    
+    ### Content Scheduler block
```
Comment (DominikaK) [inline docs/working_with_page.md]: This section duplicates publishing/advanced_publishing_options/ again. You only need a short description and a link to that page.

File: docs/creating_content_basic.md [hunk 24]
```diff
@@ -92,55 +92,6 @@ While editing, you can preview what the content will look like by clicking the *
 A Content item can have different looks for different [SiteAccesses](creating_content_advanced.md#siteaccess).
 You can select a SiteAccess to preview by using a dropdown in the preview screen.
 
-!!! enterprise
```
Comment (DominikaK) [inline docs/creating_content_basic.md]: Could use a link to the new page.

File: docs/working_with_page.md [hunk 25]
```diff
@@ -0,0 +1,207 @@
...
+    !!! tip
+        The Page Content Type contains a **Landing Page** Field Type which manages the zones and blocks.
+        Any Content item that has the **Landing Page** Field Type will behave like the Page Content Type.
+        
+    ## Creating Page
+    
+    To create a new Page, in the upper toolbar click **Content** and from the list below select **Content Structure**.
+    On the right-side toolbar click **Create** and from the list of Content items select **Landing page**.
+    In the **Page creation** select the layout and click **Create**.
+    Edit the Page title and descritpion by switching to the Field view using the [Page builder toolbar](#page-builder-toolbar).
```
Comment (DominikaK) [inline docs/working_with_page.md]:
```suggestion
    Edit the Page title and description by switching to the Field view using the [Page builder toolbar](#page-builder-toolbar).
```

File: docs/working_with_page.md [hunk 26]
```diff
@@ -0,0 +1,207 @@
...
+    You can edit it to personalize the layout and contents of your website.
+    
+    
+    !!! tip
+        The Page Content Type contains a **Landing Page** Field Type which manages the zones and blocks.
+        Any Content item that has the **Landing Page** Field Type will behave like the Page Content Type.
+        
+    ## Creating Page
+    
+    To create a new Page, in the upper toolbar click **Content** and from the list below select **Content Structure**.
```
Comment (DominikaK) [inline docs/working_with_page.md]: Maybe also mention creating a Page from the Page mode directly

File: docs/creating_content_basic.md [hunk 27]
```diff
@@ -20,7 +20,7 @@ When uploading files, the Content Type will be chosen automatically.
 
     If you are using eZ Enterprise Edition,
     you will have the *Page* Content Type available for selection.
-    Pages are edited in the Page Builder, [see below](#working-with-a-page).
+    Pages are edited in the Page Builder, [see below](working_with_page.md).
```
Comment (MagdalenaZuba) [inline docs/creating_content_basic.md]:
```suggestion
    Pages are edited in the Page Builder, [see working with page section.](working_with_page.md)
```

File: docs/working_with_page.md [hunk 28]
```diff
@@ -0,0 +1,207 @@
+# Working with a Page
```
Comment (MagdalenaZuba) [inline docs/working_with_page.md]:
```suggestion
# Working with Page
```

OR

```suggestion
# Working with the Page
```

**Comment by michalmilc90** [comment]
Implement changes in all v2 document versions.

**Pull request merged**